versioned_docs/version-3.x/reference/zmodel/import.md

```
@@ -0,0 +1,30 @@
+
+## Syntax
+
+```zmodel
+import IMPORT_SPEC
+```
+
+- *IMPORT_SPEC**: 
+
+    Path to the ZModel file to be imported. It can be:
+    
+    - An absolute path, e.g., "/path/to/user".
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)